CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# March 18, 2019
+### Rely 1.3.0
+* jUnit reporter
+* throw error when nesting top level TestFramework.describe
```

Comment:
Ocaml terminology is Raise, we should attempt to stick with that I think:

- Raise error when nesting top level describes (Nest describes like this (link to example with nested describes))